libcodechecker/libhandlers/analyze.py

```diff
@@ -177,6 +184,58 @@ def add_arguments_to_parser(parser):
                                help="File containing argument which will be "
                                     "forwarded verbatim for Clang-Tidy.")
 
+    if is_ctu_capable():
+        ctu_opts = parser.add_argument_group('cross translation unit analysis')
+        ctu_mutex_group = ctu_opts.add_mutually_exclusive_group()
+        ctu_mutex_group.add_argument('--ctu',
+                                     action='store_const',
+                                     const=[True, True],
+                                     dest='ctu_phases',
+                                     default=argparse.SUPPRESS,
+                                     help="Perform cross translation unit "
+                                          "(CTU) analysis (both collect and "
+                                          "analyze phases) using default "
+                                          "<ctu-dir> for temporary output. "
+                                          "At the end of the analysis, the "
+                                          "temporary directory is removed.")
```

Comment:
:innocent: Discontinuing the concept of "workspace" (which was the... duh... work space for both servers and analyses) in favour of using a more common `--config-dir` or `--output-dir` argument was one thing, but making 8234 separate """"output""" dirs is overkill.